Gemfile.lock

```diff
@@ -250,7 +250,7 @@ GEM
     bixby (1.0.0)
       rubocop (~> 0.50, <= 0.52.1)
       rubocop-rspec (~> 1.22, <= 1.22.2)
-    blacklight (6.20.0)
+    blacklight (6.24.0)
```

Comment:
Update from 6.20.0 to 6.24.0 largely aria and accessibility plus the span tag fix